dao/src/main/java/org/thingsboard/server/dao/sql/query/DefaultAlarmQueryRepository.java

```diff
@@ -87,6 +88,13 @@ public class DefaultAlarmQueryRepository implements AlarmQueryRepository {
             " THEN (select name from entity_view where id = a.originator_id)" +
             " END, 'Deleted') as originator_name";
```

Comment:
@ashvayka, same issue with the "Deleted" value here. Should we update this or leave this without changes?